config/boot.rb

```diff
@@ -3,3 +3,7 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
 
 require 'bundler/setup' # Set up gems listed in the Gemfile.
+
+# Bootsnap is used to speed up application load time. Enabled for development/test
+# environments.
+require 'bootsnap/setup' if %w[development test].include?(ENV['RAILS_ENV'])
```

Comment:
```suggestion
require 'bootsnap/setup' if Rails.env.development? || Rails.env.test?
```
minor suggestion.  I think these helpers make things a little clearer

Reply:
👍 Good idea

Reply:
Whoops, sorry, I initially implemented your suggestion, but I had to revert it. Turns out in `boot.rb`, we haven't actually instantiated the `Rails` singleton, so we can't use the `Rails.env` helpers yet!

Reply:
@erluti ^

Reply:
dang, i was just fooling in the console and not in the boot.rb file.  Thanks for trying it!